### PR TITLE
Add support matrix of features supported of Ceph CSI version

### PR DESCRIPTION
**Pull request by poornimag**
Signed-off-by: Poornima G <pgurusid@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
Thanks, @poornimag for the PR.

Fixes #298 

@leseb PTAL

File: README.md [hunk 1]
```diff
@@ -20,6 +20,21 @@ CephFS CSI plugins, see documentation in `docs/`.
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+Here is the matrix that describes the features supported by the different versions of Ceph CSI driver
+
+|              |        Features             | CSI driver<br>Version |Kubeernetes<br>Version|
```
Comment (Madhu-1) [inline README.md]: Kubeernetes to Kubernetes

**Comment by Madhu-1** [comment]
@poornimag can you update the table for v0.3.0 driver  also

**Comment by poornimag** [comment]
CI is failng with "./README.md:1: MD033 Inline HTML". I think that's due to the "CSI driver<br>Version", this is the only way to list two lines in a single cell in markdown table. I can get rid of the <br> and will have a long column heading. Should i do that? or allow html in readme?

**Comment by Madhu-1** [comment]
we have 2 options, disable the rule or 
use  [pure markdown](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md033---inline-html) instead  of HTML 


@poornimag will this helps https://talk.commonmark.org/t/tables-in-pure-markdown/81/145

File: README.md [hunk 2]
```diff
@@ -20,6 +20,23 @@ CephFS CSI plugins, see documentation in `docs/`.
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
```
Comment (humblec) [inline README.md]: @poornimag ideally we should have these columns :  
Branch/CSI driver Version, CSI Spec , variants like RBD and CephFS, Kubernetes version

**Comment by humblec** [comment]
@poornimag ideally we should have these columns :  
Branch/CSI driver Version, CSI Spec , variants like RBD and CephFS, Kubernetes version

**Comment by leseb** [comment]
Should wait for https://github.com/ceph/ceph-csi/pull/297

**Comment by Madhu-1** [comment]
added DNM label to avoid accidental merging

**Comment by Madhu-1** [comment]
can you send this PR to master branch?  from now onwards we will be using master for csiv1.0 development, Thanks

**Comment by wilmardo** [comment]
Also closes #268 right?

**Comment by Madhu-1** [comment]
> Also closes #268 right?

yes

**Comment by humblec** [comment]
@poornimag Can you please correct the tests?

**Comment by poornimag** [comment]
@humblec  sent another pr..., https://github.com/ceph/ceph-csi/pull/315 for master

**Pull request closed without merging**